packages/mongodb-memory-server-core/src/util/MongoBinaryDownloadUrl.ts

```diff
@@ -289,7 +289,9 @@ export class MongoBinaryDownloadUrl implements MongoBinaryDownloadUrlOpts {
       throw new UnknownVersionError(this.version);
     }
 
-    if (release >= 11 || ['unstable', 'testing'].includes(os.release)) {
+    const isTesting = ['unstable', 'testing', ''].includes(os.release);
```

Comment:
please combine this with the one from the function, to have less duplication

Reply:
I removed the function.